### ISSUE TITLE
Option to hide the statusbar icon 

### ISSUE THREAD

**Issue by Safi1012** [KrauseFx/overkill-for-mac#13]
I would love the option to hide the statusbar icon. 
I think we all know the feeling when the statusbar is becoming too cluttered ;)

**Comment by KrauseFx** [comment]
Yeah, totally agree. I'm using Bartender for that, but feel free to submit a PR adding this option to Overkill 👍

**Comment by Safi1012** [comment]
Great! Have a look at my branch ;)
https://github.com/Safi1012/overkill-for-mac

Unfortunately after hiding the menu icon, it is impossible for the user to display the app again. The user must first quit the whole application (via activity monitor) and restart the app again. I could add a prefPanel, but I think it would be an **Overkill** for such a small app. Any suggestions?

**Comment by KrauseFx** [comment]
Haha, perfect. Easy fix would be to update the README to mention Bartender (which I recommend either way) and a link to your branch?

**Comment by Safi1012** [comment]
I'm sorry, I accidentally posted the wrong link.The branch is here https://github.com/Safi1012/overkill-for-mac/tree/hide-statusbar.

**Comment by claire-cli** [comment]
heya, i know it's been almost 7 years but by any chance do you still have that branch? both links don't seem to be working anymore
